pkg/dashboard/uiserver/empty_assets_handler_windows.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package uiserver
+
+import (
+	assetfs "github.com/elazarl/go-bindata-assetfs"
+)
+
+func assetFS() *assetfs.AssetFS {
```

Comment:
Why it will not cause name conflict in non windows?

Reply:
Sorry for the late reply.

[Package build](https://golang.org/pkg/go/build/)

If a file's name, after stripping the extension and a possible _test suffix, matches any of the following patterns:
```
*_GOOS
*_GOARCH
*_GOOS_GOARCH
```
(example: source_windows_amd64.go) where GOOS and GOARCH represent any known operating system and architecture values respectively, then the file is considered to have an implicit build constraint requiring those terms (in addition to any explicit constraints in the file).